### PR TITLE
changed kwargs of Normal_ZeroCorr-constructor

### PR DESCRIPTION
**Pull request by mgrub**
- naming convention according to scipy.stats.norm
- changed all occurences of Normal_ZeroCorr() to the new loc/scale-kwargs
- resolves issue #40

### CODE REVIEW

**Comment by eichstaedtPTB** [comment]
Use of loc and scale in the methods itself, though, is missing. There still mean and cov are used.

File: PyDynamic/uncertainty/propagate_MonteCarlo.py [hunk 1]
```diff
@@ -26,318 +26,320 @@
...
+    def __init__(self, loc=None, scale=None):
+        """
+        Parameters
+        ----------
+            loc: np.ndarray, optional
+                mean values, default is zero
+            scale: np.ndarray, optional
+                standard deviations for the elements in loc, default is zero
+        """
+        if isinstance(loc, np.ndarray):
```
Comment (mgrub) [inline PyDynamic/uncertainty/propagate_MonteCarlo.py]: We should reduce checks to either s
- isinstance log np.ndarry OR isinstance scale nd.array
- ifnboth ndarray, check length
-turn scalar intonvecotr of appropriate length

File: PyDynamic/uncertainty/propagate_MonteCarlo.py [hunk 2]
```diff
@@ -26,318 +26,320 @@
...
+            self.mean = loc
```
Comment (mgrub) [inline PyDynamic/uncertainty/propagate_MonteCarlo.py]: Rename self.mean ->self.loc
Same with scale

Also check for uses of both whereever class is called

File: test/test_propagate_MonteCarlo.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+""" Perform tests on the method *make_equidistant*."""
+
+import numpy as np
+from pytest import raises
+
+import PyDynamic.uncertainty.propagate_MonteCarlo as mc
+
+
+def test_Normal_ZeroCorr_constructor():
+    N = 10
```
Comment (mgrub) [inline test/test_propagate_MonteCarlo.py]: Adapt tests tomchanges in constructor

Make test more modular

**Pull request merged**